src/middleware/validateSchema.js

```diff
@@ -32,13 +52,23 @@ module.exports = (schema ) => {
 						// fetch only message and type from each error
 						message: map(error.details, ({ message }) => (
 							message.replace(/['"]/g, '')
+								.replace('body.', '')
+								.replace('query.', '')
+								.replace('params.', '')
 						)).join(', ')
 					}
 				}
 
+
 				return res.status(400)
 					.json( JoiError )
 			}
+
+			// reassign validated data to request
+			req.body = strippedReq.body
```

Comment:
really awesome work so far. 

with this part, there's a little more to it, the validated data is from the validation joi does not the stripped request

so line 43 will rather be

```js
const { value, error } = 
```

so its the value that is the validated data. after joi has done its own stuff. like strip unwanted keys, add defaults and so on.

and this part will be something like 

```suggestion
			req.body = value.body || strippedReq.body
```


 the short circuit is just in case the body isn't what was being validated in the schema. In that case, we still preserve the original

Kindly let me know if anything

Reply:
Thanks. I have updated it @jocrah

Reply:
cool @itzgeebee, checking it out